### ISSUE TITLE
Issue on producing the similar result in my own dataset

### ISSUE THREAD

**Issue by Jacobieee** [shruti-jadon/Covid-19-Detection#2]
Hello, thanks for your contribution, I like your work very much!

I'm currently trying to classify chest X-ray images into 4 classes. There are  1372 for train and 560 for test and the numbers of images in each class are the same in both training and testing set . I tried to use siamese networks to extract features, PCA+tsne to decrease the embedding dimension, and k-means and Gaussian mixture for classification. And I got a very poor classification result.
![截屏2021-07-19 上午2 47 27](https://user-images.githubusercontent.com/41676649/126380359-991017c1-e1d4-40b0-b9ef-5ba6f48b14a0.png)
And I think training is doing good.
![截屏2021-07-21 上午2 10 41](https://user-images.githubusercontent.com/41676649/126380534-6e9b0bb4-d429-459e-827a-c7a67641bfe1.png)
I think my dataset is very similar to yours so I think such a poor result is very weird. Could you provide me with some instructions for improving the model? I will appreciate so much for your help.

**Comment by shruti-jadon** [comment]
Interesting, I will suggest increasing the number of epochs, do some data
augmentation to increase data size, and also observe both Val loss and
train loss. If both are less that means your amount of data is less causing
underfitting if Val loss is high and train loss is low that means
overfitting in that case tune down the number of params in siamese network
architecture and experiment further.

Thanks,
Shruti Jadon
